testing/e2e/cluster_supply_chain_test.go

```diff
@@ -29,14 +30,49 @@ import (
 func TestClusterSupplyChain(t *testing.T) {
 	testSuite := it.CommandLineIntegrationTestSuite{
 		{
-			Name:                      "List the existing supply chains",
-			Command:                   *it.NewTanzuAppsCommandLine("cluster-supply-chain", "list"),
-			ExpectedCommandLineOutput: it.GetFileAsString(t, filepath.Join(it.ConsoleOutBasePath, "list-csc", "test-list-csc.txt")),
+			Name:    "List the existing supply chains",
+			Command: *it.NewTanzuAppsCommandLine("cluster-supply-chain", "list"),
+			Verify: func(t *testing.T, output string, err error) {
+				match, err := regexp.MatchString(`.*.\nci-test[ ]{3,}[<>a-zA-Z]{5,}[ ]{3,}[0-9]{1,}s\n.*`, output)
+				if err != nil {
+					t.Error("Error while validating the output", err)
+					t.FailNow()
+				}
+				if !match {
+					t.Error("Expected 'ci-test   <status>   0s' to be present in the output")
```

Comment:
include the actual output you received in the command as well for the error

Reply:
Added to the error message